File: CC_api/Models/Conversation.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace CC_api.Models
{
  public class Conversation
  {
    [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
    public int conversationid { get; set; }
    //public string ConversationName { get; set; }
    //public string ConvoDesc { get; set; }
    public int adscompanyid { get; set; }//ad requesting company name
    public string company_name { get; set; }
    public string user_name { get; set; }
    public int company_id { get; set; }
    public int is_active { get; set; }
    public int user_id { get; set; }
    public int negotiation_id { get; set; }
    public string company_logo { get; set; }

    public string negotiator_company_name { get; set; }

    public string negotiator_company_logo { get; set; }

  }
}

File: cc-ui/src/app/DTO/conversation.ts
export interface conversation {
  adcompanyId:number;  
  conversationid: number;
  company_id: number;
  user_id:number;
  company_name:string;
  first_name:string;
  is_active:number;
  negotiation_id:number;
  company_logo:string;
  negotiator_company_name:string;
  negotiator_company_logo:string;
}
export interface Candidatee{
  user_id:number,
  company_id:number,
  first_name:string,
  last_name:string,
  designation:string
  selected:boolean
  company_name: string
}
      